ows_geopackage_extension.md

```diff
@@ -0,0 +1,109 @@
+# GeoPackage 1.0 Extension
+
+**DRAFT**
+
+Extension follows template from Annex I of the OGC [GeoPackage 1.0 Specification](http://www.geopackage.org/).
+
+## Extension Title
+
+OWS Context
+
+## Introduction
+
+Goal: Store context of a mapping project including styling and resources in a single GeoPackage file.
+
+Approach: introducing an OWS-context that defines a set of layers on a set of data tables using certain styles. Any data table can be referenced by the OWS context multiple times, table ows_style and ows_style-reference define which styles are available for that layer. Resources from ows_resources can be referenced to from metadata, styles and context, to enable inclusion of images, pdf, etc.
+
+OWS-context 1.0 defines the capability to reference layers from OWS services and certain filetypes (gml/kml). In this scenario we introduce the capability to reference tables local to the Geopackage.
+
+The ows-context specification is extended with an offeringtype="gpkg" and OWS-context is extended to support referencing resources local to the Geopackage by using #table={table}[&{fieldname}={value}].
+
+```
+<owc:offering
+ code="http://www.opengis.net/spec/owc-atom/1.0/req/gpkg">
+ <owc:content type="application/x-sqlite" href="#table=MyPoints" />
+ <owc:styleSet>
+ <owc:name>simple_point</owc:name>
+ <owc:title>Simple point</owc:title>
+ <owc:content href="#table=ows_style&name=simplePoint" type="application/sld+xml"/>
+</owc:styleSet>
+</owc:offering>
+```
+
+
+
+## Extension Author
+
+Pirmin Kalberer, author_name `pka`.
```

Comment:
You can remove my name, since my contribution is too little for mentioning, but thanks!